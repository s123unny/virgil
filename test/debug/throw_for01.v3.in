b ../core/throw_for01.v3 5
run
q
