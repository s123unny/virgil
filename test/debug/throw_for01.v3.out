breakpoint|../core/throw_for01.v3 5|-1
stop|end
result|!DivideByZeroException
	in throw_for01.main() [../core/throw_for01.v3 @ 4:26]

